rules-reviewed/eap6/uncategorized/environment-dependent.windup.xml

```diff
@@ -23,7 +23,7 @@
                 </javaclass>
             </when>
             <perform>
-                 <hint title="Dynamic class instantiation" effort="0" category-id="optional">
+                 <hint title="Dynamic class instantiation" effort="1" category-id="optional">
```

Comment:
We've seen this change proposed several times, and can't merge it as things stand -- on large apps it can trigger thousands of times, and make up >90% of the story point estimate. We can reopen the topic of how to handle issues like this (like perhaps by having options for effort scaling other than [constant * #incidents]) but until then we can't attach effort to this hint.